cleverhans/experimental/certification/utils.py

```diff
@@ -10,10 +10,8 @@
 
 def diag(diag_elements):
   """Function to create tensorflow diagonal matrix with input diagonal entries.
-
   Args:
```

Comment:
Usually we leave blank space between function description and list of arguments, as well as between list of arguments and return value.
But it seems like most of the blank lines separating logical blocks in the comments were removed in this file

As an example of how function comments should be formatted in python, take a look at this piece of TensorFlow code: https://github.com/tensorflow/tensorflow/blob/7285f3fbddf2054c3971973f62f998579d00d8ec/tensorflow/python/layers/convolutional.py#L122

Overall idea is following:
```python
def some_function(arg1, arg2, arg3, ...):
  """Short one-line description.

  Optional multi-line description

  Args:
    arg1: description of arg1
    arg2: decsription of arg2, if you need to split this description
      into multiple lines then each subsequent line should have proper
      indentation
    arg3: ...
    ...

  Returns:
    description of return value

  Raises:
    ValueError: description in what case ValueError is raised
    SomeOtherError: ...
    ...
  """
  # function body

```